src/Microsoft.PowerShell.Commands.Management/commands/management/GetClipboardCommand.cs

```diff
@@ -13,7 +14,9 @@
 using System.Management.Automation;
 using System.Media;
 using System.Runtime.InteropServices;
+#if !UNIX
 using System.Windows.Forms;
+#endif
```

Comment:
What about ARM? In csproj we exclude it.

Reply:
That's a good point